Sources/MultipartKit/FormDataEncoder/FormDataEncoder.swift

```diff
@@ -26,6 +27,20 @@ public struct FormDataEncoder: Sendable {
     public func encode<E: Encodable>(_ encodable: E, boundary: String) throws -> String {
         try MultipartSerializer().serialize(parts: parts(from: encodable), boundary: boundary)
     }
+    
+    /// Encodes an `Encodable` item to `Data` using the supplied boundary.
+    ///
+    ///     let a = Foo(string: "a", int: 42, double: 3.14, array: [1, 2, 3])
+    ///     let data = try FormDataEncoder().encodeToData(a, boundary: "123")
+    ///
+    /// - parameters:
+    ///     - encodable: Generic `Encodable` item.
+    ///     - boundary: Multipart boundary to use for encoding. This must not appear anywhere in the encoded data.
+    /// - throws: Any errors encoding the model with `Codable` or serializing the data.
+    /// - returns: `multipart/form-data`-encoded `String`.
+    public func encodeToData<E: Encodable>(_ encodable: E, boundary: String) throws -> Data {
```

Comment:
```suggestion
    public func encodeToData(_ encodable: some Encodable, boundary: String) throws -> Data {
```
Since the package requires Swift 5.7, we can use the more concise form of the generic type here. (If only the syntax was flexible enough to do the same with the decode side of things 😅). The `<E: Encodable>` and `some Encodable` forms are so synonymous that switching between them is not a source compatibility break. (In fact, can you make the same change in the `String` version of the method above?)